### ISSUE TITLE
Changing header text on item click not working

### ISSUE THREAD

**Issue by elmokadim** [afollestad/sectioned-recyclerview#45]
- Library version: `0.4.1`
 - Device OS version: `7.0.`
 - Device Manufacturer: `Samsung`
 - Device Name: `Galaxy S7 Edge`

```java
@Override
  public void onBindHeaderViewHolder(SectionedViewHolder holder, int section, boolean expanded) {
    String sectionName = services.get(section).getCat();
    HeaderVH headerVH = (HeaderVH) holder;
    headerVH.tvTitle.setText(("-   " +  sectionName));
    headerVH.itemView.setOnClickListener(v -> {
      toggleSectionExpanded(section);
      if (expanded){
        headerVH.tvTitle.setText(("+   " +  sectionName));
      } else {
        headerVH.tvTitle.setText(("-   " +  sectionName));
      }
      notifySectionChanged(section);
    });
  }
```
When trying to change header text on item click the title remains the same, So I added notifySectionChanged(section); but not working also.

Also I wanna know how to start list collapsed

**Comment by afollestad** [comment]
Fixed in 0.4.3

**Comment by elmokadim** [comment]
Thanks
